docs/src/pages/customization/themes/themes.md

```diff
@@ -216,13 +216,62 @@ If you want to learn more about typography, you can check out [the typography se
 
 {{"demo": "pages/customization/themes/TypographyTheme.js"}}
 
-#### Typography - Font family
+### Typography - Font family
```

Comment:
I don't believe we repeat the heading from the level above like this elsewhere in the docs. Would this be a good opportunity remove it, or is the extra clarity needed here for some reason? cc: @oliviertassinari

Reply:
Great idea.

Reply:
@johnrichter what do you think? Do you want to change the headers?

Reply:
I agree. Certainly. One moment

Reply:
Ready to go 👍